documentation/prototype-usage-instructions.md

```diff
@@ -134,12 +134,12 @@ key as shown below.
 
 At the moment this isn't transferred well by the derivation algorithm
 from PDF to HTML, but this area will see improvements. (The PDF structure
-is already correct, as one cane see in ngPDF, but the HTML/CSS
-styling currently loses the grid layout.)
+is already correct, as one can see in ngPDF, but the HTML/CSS
+styling currently looses the grid layout.)
```

Comment:
```suggestion
styling currently loses the grid layout.)
```

Reply:
:-)your lovely language at work ... we German's would always expect that it needs two "o"s